packages/teams-js/src/public/app.ts

```diff
@@ -470,6 +470,14 @@ export namespace app {
             // After Teams updates its client code, we can remove this default code.
             try {
               const givenRuntimeConfig: IRuntime = JSON.parse(runtimeConfig);
+              // Check that givenRuntimeConfig is a valid instance of IRuntimeConfig
+              try {
+                if (!givenRuntimeConfig.apiVersion) {
+                  throw new Error('Received runtime config is invalid');
+                }
+              } catch {
+                throw new Error('Received runtime config is invalid');
+              }
```

Comment:
Why the catch and re-throw?

Reply:
Won't it be caught in the existing catch?

Reply:
If you change the check to `if (!givenRuntimeConfig || !givenRuntimeConfig.apiVersion)` then you don't need to put the check in its own try-catch.